### PR TITLE
Course complete

### PR DESCRIPTION
**Pull request by sujoydatta**

### CODE REVIEW

File: app/controllers/supervisor/courses_controller.rb [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+
+  def index
+    @courses= Course.all.paginate page: params[:page], per_page: 10
+  end
+
+  def show
+  end
+
+  def new
+    @course= Course.new
```
Comment (chuanhtuan) [inline app/controllers/supervisor/courses_controller.rb]: space before =

File: app/controllers/supervisor/courses_controller.rb [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+  end
+
+  def create
+    @course= Course.new course_params
```
Comment (chuanhtuan) [inline app/controllers/supervisor/courses_controller.rb]: sapce

File: app/controllers/supervisor/courses_controller.rb [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+    if @course.save
+      flash[:success] = t 'course_creation_text'
+      render :show
```
Comment (chuanhtuan) [inline app/controllers/supervisor/courses_controller.rb]: redirect @course

File: app/views/supervisor/courses/_course.html.erb [hunk 4]
```diff
@@ -0,0 +1,25 @@
...
+          <font color: "red">
+            <span>
+              <%= link_to t(:read_more), [:supervisor, course] %>
+            </span>
+          </font>
+        <% else %>
+          <%= course.description %>
+        <% end %>
+    </td>
+    <td><%= l course.start_date, :format => :short %> </td>
```
Comment (chuanhtuan) [inline app/views/supervisor/courses/_course.html.erb]: format: :short

File: app/controllers/supervisor/courses_controller.rb [hunk 5]
```diff
@@ -0,0 +1,51 @@
+class Supervisor::CoursesController < ApplicationController
+  before_action :load_course, except: [:index, :new, :create]
+
+  def index
+    @courses= Course.all.paginate page: params[:page], per_page: 10
```
Comment (chuanhtuan) [inline app/controllers/supervisor/courses_controller.rb]: add 1 space before '='

Comment (sujoydatta) [inline app/controllers/supervisor/courses_controller.rb]: We will modify it in next pull request

**Pull request merged**